From: Jeremy Kerr <jk@ozlabs.org>

For boot wrapper debugging use only.  Some versions of debian binutils
choke on this.

 arch/powerpc/boot/wrapper |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/powerpc/boot/wrapper	2007-08-27 14:01:20.000000000 -0400
+++ b/arch/powerpc/boot/wrapper	2007-08-27 14:01:21.000000000 -0400
@@ -216,6 +216,7 @@ fi
 
 if [ "$platform" != "miboot" ]; then
     ${CROSS}ld -m elf32ppc -T $lds -o "$ofile" \
+	-Map "$ofile.map" \
 	$platformo $tmp $object/wrapper.a
     rm $tmp
 fi
